src/main/scala/com/github/codelionx/distod/types/OrderDependency.scala

```diff
@@ -50,7 +50,7 @@ object OrderDependency {
           val attribute1Name = names(attribute1)
           val attribute2Name = names(attribute2)
           val contextNames = context.map(names).mkString(", ")
-          s"{$contextNames}: $attribute1Name ~ $attribute2Name$orderOfSecondAttribute"
+          s"{$contextNames}: $attribute1Name↑ ~ $attribute2Name$orderOfSecondAttribute"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Non ascii characters are not allowed](https://app.codacy.com/app/CodeLionX/distod/pullRequest?prid=4581953)